### PR TITLE
v2.3.0 Blog Post

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

File: input/blog/2022-10-14-cake-v2.3.0-released.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+- [__#3880__](https://github.com/cake-build/cake/issues/3880) Better support global script cache.
+- [__#2953__](https://github.com/cake-build/cake/issues/2953) Allow setting MSBuild target via MSBuildSettings using a string.
+- [__#2591__](https://github.com/cake-build/cake/issues/2591) Extensibility issue - CakeTaskBuilder is sealed and CakeTaskBuilder(CakeTask task) is internal. .
+
+__Bug__
+
+- [__#3931__](https://github.com/cake-build/cake/issues/3931) Cake fails to load native libraries on Ubuntu 22.04.
+- [__#3894__](https://github.com/cake-build/cake/issues/3894) Guard against null Console instance on InfoFeature.
+- [__#3879__](https://github.com/cake-build/cake/issues/3879) Build script caching throws after running dry-run on non-changed Cake script.
+- [__#3878__](https://github.com/cake-build/cake/issues/3878) OpenCover filters should be case sensitive .
```
Comment (pascalberger) [inline input/blog/2022-10-14-cake-v2.3.0-released.md:210]:
```suggestion
- [__#3878__](https://github.com/cake-build/cake/issues/3878) OpenCover filters should be case sensitive.
```

**Review by pascalberger — APPROVED** [review]
LGTM beside some typos

**Review by pascalberger — APPROVED** [review]
LGTM

**Review by patriksvensson — APPROVED** [review]
LGTM!

**Review by bjorkstromm — APPROVED** [review]
Lovely! I'm going to use `Command` a lot :)

**Pull request merged**